FoodBookApp/UI/Views/Settings/SettingsView.swift

```diff
@@ -54,12 +57,17 @@ struct SettingsView: View {
                             UserDefaults.standard.set(lunchTime ? true : false, forKey: "sendLunchTimeNotification")
                         }
                 }
-                // TODO: @maria
+                
+                // MARK: - Authentication Settings
                 Section(header: Text("AUTHENTICATION")) {
+                    Toggle("Authenticate before uploading reviews", isOn: $authenticateReviews)
+                        .onChange(of: authenticateReviews) {
+                            UserDefaults.standard.set(authenticateReviews, forKey: "authenticateBeforeUploading")
+                        }
                 }
             }
-            
             Spacer()
+            Text("auth \(authenticateReviews ? "yes" : "no")")
```

Comment:
Is this something for tests?

Reply:
yepp, i just deleted it hehe i forgot to do so ealier

Reply:
i'll commit with other changes